src/transactions/utils/format.js

```diff
@@ -18,3 +18,12 @@ export const prependPlusToPublicKeys = publicKeys =>
 
 export const prependMinusToPublicKeys = publicKeys =>
 	publicKeys.map(publicKey => `-${publicKey}`);
+
+export const validateKeysgroup = keysgroup => {
```

Comment:
Does this function really belong in a file named `format.js`? Maybe we should have a module for validation.